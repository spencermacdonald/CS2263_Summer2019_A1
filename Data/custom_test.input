u 25
u 26
o
u 60
e
x