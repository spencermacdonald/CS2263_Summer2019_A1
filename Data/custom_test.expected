25
26
26
60
60
Successfully executed 5 instructions
|_ 25 _|
|_ 60 _|
2 elements